yb-voyager/cmd/importData.go

```diff
@@ -1104,8 +1104,6 @@ func init() {
 
 	registerCommonGlobalFlags(importDataCmd)
 	registerCommonGlobalFlags(importDataToTargetCmd)
-	registerCommonImportFlags(importDataCmd)
```

Comment:
How come we're getting rid of "continue-on-error"?

Reply:
Yes.